src/Orchard/Mvc/ModelBinders/BooleanBinderProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Web.Mvc;
+
+namespace Orchard.Mvc.ModelBinders {
+    public class BooleanBinderProvider : IModelBinderProvider, IModelBinder {
+
+        public IEnumerable<ModelBinderDescriptor> GetModelBinders() {
+            return new[] {
+                             new ModelBinderDescriptor {
+                                                           ModelBinder = this,
+                                                           Type = typeof(bool)
+                                                       }
+                         };
+        }
+
+        public object BindModel(ControllerContext controllerContext, ModelBindingContext bindingContext) {
+            var value = false;
+            try {
+                value = (bool)bindingContext.ValueProvider.GetValue(bindingContext.ModelName).ConvertTo(typeof(bool));
```

Comment:
I assume this is for the potential null ref I mentioned. Usually I use something like `Convert.ToBoolean( /* something that can be null */)`

```suggestion
                value = Convert.ToBoolean(bindingContext.ValueProvider.GetValue(bindingContext.ModelName));
```